github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -168,6 +168,35 @@ postsubmits:
                 memory: "8Gi"
             securityContext:
               privileged: true
+    - name: publish-github-jira-proxy-image
```

Comment:
Great to have the publish! Along the way we should also have a presubmit job that tests whether the image is buildable, see an example here: https://github.com/kubevirt/project-infra/blob/e25b96d539620be729da0617d5430345da89c2d6/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml#L141